lib/walmart_tracking_retriever.py

```diff
@@ -0,0 +1,48 @@
+import re
+from typing import Tuple, Optional, List
+
+from lib.email_tracking_retriever import EmailTrackingRetriever
+
+
+class WalmartTrackingRetriever(EmailTrackingRetriever):
+
+  tracking_regexes = [
+      r'<a\s*href="(.*)\s*(.*)\s*<span class="columnHeader.*\s*(.*)<\/a>'
```

Comment:
Need help with this portion

Reply:
@gbrodman Think you can help? Not so great with regex. Let me know I can provide you the html of this portion in the email if need be

Reply:
cc @CydeWeys